packages/cosmic-swingset/bin/ag-nchainz

```diff
@@ -20,9 +20,11 @@ real0=$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")
 thisdir=$(cd "$(dirname -- "$real0")" && pwd -P)
 
 PATHCONFIG=nchainz/config/paths
-CLI=ag-cosmos-helper
-DAEMON=ag-chain-cosmos
-SOLO=$thisdir/../solo/bin/ag-solo
+DAEMON=agd
+DAEMON_CMD="$thisdir/../../golang/cosmos/build/$DAEMON"
+CLI="$DAEMON_CMD"
```

Comment:
`CLI` changed to absolute? I'm curious why.

Reply:
Before, it required things in the `$PATH`, but almost all of the rest of the SDK was changed to use the absolute filename to prevent needing stuff in the `$PATH`.  `ag-nchainz` was left behind; it didn't get updated when the rest of the SDK did.  I didn't want it to languish anymore.